Hiren patch 1

hiren1612 opened the pull request:
Resolves #1, Resolves #2, Resolves #3, Resolves #4

tevyaw committed:
Update readme.txt

Updating readme.txt for new 3.0 version with name change to "Latter-day Linker", removal of "passage search" option, etc. Removed lots of line breaks where there shouldn't have been. Cleaned up text a bunch to reflect new name, etc.

tevyaw committed:
Update ldslinker.php

Changed name. Updated various other meta data.

hiren1612 committed:
issues solved.

hiren1612 committed:
renaming the plugin to Latter-day Linker

hiren1612 commented:
Renamed plugin!

tevyaw commented:
@hiren1612 so will the renaming break backward compatibility? Or will old settings still work in the new version? I just don't want people to update and then find their settings are messed up.

tevyaw commented:
Oh, and one more thing: could we just remove the whole "URL construction method" setting? It's not longer necessary, since there's only one option, so just remove it so as not to confuse anyone.

tevyaw committed:
Update readme.txt

Updated the version 3.0 changelog for clairity, updated one FAQ item so it no longer points to Joey's contact form.
